src/epub.rs

```diff
@@ -51,9 +51,11 @@ impl Epub {
     fn get_chapters(&mut self, spine: Vec<(String, String)>) {
         for (title, path) in spine {
             let xml = self.get_text(&format!("{}{}", self.rootdir, path));
+            let opt = ParsingOptions { allow_dtd: true };
             // https://github.com/RazrFalcon/roxmltree/issues/12
             // UnknownEntityReference for HTML entities
-            let doc = Document::parse(&xml).unwrap();
+            let doc = Document::parse_with_options(&xml, opt)
+                .unwrap_or_else(|_| panic!("Could not parse path: {}", &path));
```

Comment:
`unwrap_or_else ... panic` doesn't look idiomatic, maybe an `expect` is better, or use `process::exit` like in main

Reply:
I think it is idiomatic way if you don't want to return Result. I haven't changed to Result bc you modify in-place.

- https://stackoverflow.com/questions/55513198/is-there-a-more-concise-way-to-format-expect-message
- https://www.reddit.com/r/rust/comments/7hxfkr/lazily_constructing_an_expect_message/

Reply:
i don't want to introduce a new idiom in a single place. i'll accept but change it. you can make another pr which focuses on error handling.